src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -931,9 +931,11 @@ public async Task<AzureDevOpsReleaseDefinition> RemoveAllArtifactSourcesAsync(st
         /// <param name="accountName">Azure DevOps account name</param>
         /// <param name="projectName">Project name</param>
         /// <param name="releaseDefinition">Release definition to be updated</param>
-        public async Task<int> StartNewReleaseAsync(string accountName, string projectName, AzureDevOpsReleaseDefinition releaseDefinition, int barBuildId)
+        /// <param name="barBuildId">BAR BuildID to be passed as variable to the pipeline</param>
+        /// <param name="channelClassification">BAR Channel Classification to be passed as variable to the pipeline</param>
+        public async Task<int> StartNewReleaseAsync(string accountName, string projectName, AzureDevOpsReleaseDefinition releaseDefinition, int barBuildId, string channelClassification = null)
         {
-            var body = $"{{ \"definitionId\": {releaseDefinition.Id}, \"variables\": {{ \"BARBuildId\": {{ \"value\": \"{barBuildId}\" }} }} }}";
+            var body = $"{{ \"definitionId\": {releaseDefinition.Id}, \"variables\": {{ \"BARBuildId\": {{ \"value\": \"{barBuildId}\" }}, \"ChannelClassification\": {{ \"value\": \"{channelClassification}\" }} }} }}";
```

Comment:
Maintaining a JSON string is not super escalable and could lead to mistakes. I'd suggest to create an object that then you serialize to string and use that instead. [This](https://github.com/dotnet/arcade-services/blob/4362345e236509143a7e3af380b3dc47c6b91f8e/src/Microsoft.DotNet.Darc/src/DarcLib/GitHubClient.cs#L123) method does that.